src/content/web3/index.md

```diff
@@ -80,6 +80,21 @@ On Web3, your data lives on the blockchain. When you decide to leave a platform,
 
 Web 2.0 requires content creators to trust platforms not to change the rules, but censorship resistance is a native feature of a Web3 platform.
 
+#### Decentralized autonomous organizations (DAOs) {#daos}
+
+As well as owning your data in Web3, you can own the platform as a collective, using tokens that act like shares in a company. DAOs let you coordinate decentralized ownership of a platform and make decisions about its future.
```

Comment:
I think given the start of this sentence `As well as owning your data in Web3` this might be better located under the `Ownership` section, but not a strong stance. Overall looks good to me

Reply:
> you can own the platform as a collective

I assume you're referring to cases where a platform tokenizes and distributes its governance rights, such as ENS? Not sure I like restricting this to just "platforms." With the private keys and programmable money that web3 comes with we can coordinate on 

```suggestion
DAOs involve shared ownership and decision making power over assets. Smart contracts determine the rules for these assets, and typically involve voting mechanisms for token or share holders to make decisions on spending. Many defi platforms are governed by DAOs which enables users to participate in distributed decision making.
```

Not sure how this would fit into everything else, but just a quick brain dump